Integrations/integration-ThreatX.yml

```diff
@@ -0,0 +1,692 @@
+
+    @logger
+    def get_entities_command():
+        entity_name = demisto.args().get('entity_name', None)
+        entity_id = demisto.args().get('entity_id', None)
+        entity_ip = demisto.args().get('entity_ip', None)
+        results = get_entities(entity_name, entity_id, entity_ip)
+        md = []
+        dbot_scores = []
+        ip_enrich = []
+        output = []
+
+        for entity in results:
+            # Grab the entity risk so we can set the Dbot score for the Actor IPs
+            e_id = entity['id']
+            e_risk = get_entity_risk(e_id)
+
+            if isinstance(e_risk, list):
+                if 'risk' in e_risk[-1]:
+                    e_risk = e_risk[-1]['risk']
+                else:
+                    e_risk = 0
+            else:
+                e_risk = 0
+
+            entity['risk'] = e_risk
+
+            iplist = []
+
+            if 'actors' in entity:
+                for actor in entity['actors']:
+                    # Convert decimal IP to dotted quad
+                    if 'ip_address' in actor:
+                        ipdot = pretty_ip(actor['ip_address'])
+                        iplist.append(ipdot)
+                        actor['ip_address'] = ipdot
+
+                    # Convert interval_time_start, interval_time_stop, fingerprint.last_seen to pretty time format
+                    try:
+                        actor['interval_time_start'] = pretty_time(actor['interval_time_start'])
+                    except:
+                        pass
+
+                    try:
+                        actor['interval_time_stop'] = pretty_time(actor['interval_time_stop'])
+                    except:
+                        pass
+
+                    try:
+                        actor['fingerprint']['last_seen'] = pretty_time(actor['fingerprint']['last_seen'])
+                    except:
+                        pass
+
+                    # Set Dbot score
+                    dbscore = set_dbot_score(e_risk)
+
+                    dbot_scores.append({
+                        'Vendor' : 'ThreatX',
+                        'Indicator' : ipdot,
+                        'Type' : 'ip',
+                        'Score' : dbscore
+                    })
+
+                    if dbscore == 3:
+                        ip_enrich.append({
+                            'Malicious': {
+                                'Data': ipdot,
+                                'Vendor': 'ThreatX',
+                                'Description': 'ThreatX risk score is ' + str(e_risk)
+                            }
+                        })
+
+            md.append({'Name': entity['codename'],
+                       'ID': entity['id'],
+                       'IP Addresses': ', '.join(iplist),
+                       'ThreatX Risk Score': e_risk,
+                       'DBot Score': dbscore
+                       })
+
+            ec = {
+                'Threatx.Entity(val.ID && val.ID == obj.ID)': {
```

Comment:
Please use the `===` operator.

Reply:
done - I changed this for all of the javascript areas. Let me know if you only wanted it for this particular line.